TARS/src/common/packet.h

```diff
@@ -61,9 +82,9 @@ struct LowGData {
     float gx;
     float gy;
     float gz;
```

Comment:
(also the three lines above this, and the lines in struct HighGData) 
Might as well convert to use the Acceleration and Gyroscope structs